mavros/ff581a0/ff581a0.bug

```diff
@@ -60,3 +60,9 @@ bugzoo:
   fix-commit: ff581a0752107f7ad875c16f83f7b75ccd92f847
   fork-urls:
     - https://github.com/robust-rosin/mavros
+fault-codes:
+  - MODELS:ROBOT
+  - MODELS:TRANSFORMATIONS
+failure-codes:
```

Comment:
From a quick glance, it like the `fault-codes` and `failure-codes` are identical to one another for all of the bugs?

Reply:
you are right, var mix-up in the script.